lib/source/pl/lib/std/math.cpp

```diff
@@ -4,9 +4,23 @@
 #include <pl/core/log_console.hpp>
 #include <pl/core/evaluator.hpp>
 #include <pl/patterns/pattern.hpp>
+#include <pl/helpers/buffer.hpp>
+#include <numeric>
```

Comment:
Please use `<>` for all includes and separate regular includes from stdlib includes